addons/stock/models/stock_move.py

```diff
@@ -1523,20 +1523,26 @@ def _split(self, qty, restrict_partner_id=False):
         return new_move.id
 
     def _recompute_state(self):
```

Comment:
Maybe only backport this one: https://github.com/odoo/odoo/commit/70c721e5a21659977ea58631a7c424a7e8555849#diff-55c6314416a6a400da6acd5018d161a55eeeb0e3008fec8828121e3dd12be0ebL1591

Reply:
Oh yes I missed it.